lkft-fastboot.jinja2

```diff
@@ -14,3 +14,41 @@
   series: lkft
   email-notification: {{CI_MAIL_RECIPIENTS | default('""')}}
 {% endblock metadata %}
+
+{% block deploy_target %}
+{{ super() }}
+- test:
+    namespace: tlxc
+    timeout:
+      minutes: 30
+    definitions:
+    - from: inline
+      name: kir
+      path: inline/kir.yaml
+      repository:
+        metadata:
+          description: Squash kernel, dtb and modules into rootfs
+          format: Lava-Test Test Definition 1.0
+          name: resize-rootfs
+        run:
+          steps:
+          - pwd
+          - cd /lava-lxc
+          - git clone -b 20191120 https://github.com/linaro/kir.git
+          - ./kir/lava/board_setup.sh {{DEVICE_TYPE}}
+
+- deploy:
+    timeout:
+      minutes: 40
```

Comment:
can we have the timeout defined externally with variable? This way we can calculate total job timeout.